src/_locales/de/messages.json

```diff
@@ -36,15 +36,15 @@
         "description": "Shown in the popup on disabled sites."
     },
     "popup_disable_for_site": {
-        "message": "Für diese Site deaktivieren",
+        "message": "Für diese Webseite deaktivieren",
         "description": "Button in the popup."
     },
     "popup_enable_for_site": {
-        "message": "Für diese Site aktivieren",
+        "message": "Für diese Webseite aktivieren",
```

Comment:
Same, the non-breaking spaces should stay.

Reply:
Used Seite and put back the non-breaking spaces